### ISSUE TITLE
Inquiry About Parameter Settings for Realsense camera_type

### ISSUE THREAD

**Issue by YukiFan** [iSEE-Laboratory/EconomicGrasp#5]
I attempted to run the code with camera_type set to Realsense and used the parameter settings that designed for the Kinect, which include:
###  parser.add_argument('--weight_decay', type=float, default=0., help='Optimization L2 weight decay [default: 0]')

after several iterations of training, I encountered an issue where the model starts producing tensors filled with zeros, 
For instance, when I print cur_mask, I see
tensor([False, False, False, ..., False, False, False], device='cuda:0')
As a result, the output of cur_feat is an empty tensor:
tensor([], device='cuda:0', size=(0, 512), grad_fn=<IndexBackward0>)

so could you please share your Parameter Settings for Realsense camera_type?
Thanks for your reply~

**Comment by DravenALG** [comment]
Hi, in fact, I just train my model in both Kinect and Realsense cameras using the same parameters.

Could you provide more details about your training?

**Comment by YukiFan** [comment]
Thank you for your reply. Could it be an issue with my environment? After I changed the learning_rate in the code to 0.0001, the code can now run    >_<

**Comment by DravenALG** [comment]
I think there is possible that it is due to the environment. I have also encountered the cur_mask issue before (no valid points after mask), and I fixed it by reinstall all the libs about the environment.

You can also try the 0.0001 lr to see what would happen. Good luck.

**Comment by YukiFan** [comment]
thanks~~
